src/js/views/patients/patient/sidebar/sidebar_views.js

```diff
@@ -146,6 +146,27 @@ const sidebarWidgets = {
       };
     },
   },
+  fieldWidget: {
+    className: 'widgets-value',
+    template: hbs`{{ displayValue }}{{#unless displayValue}}{{{ defaultHtml }}}{{/unless}}`,
+    templateContext() {
+      const fields = this.model.getFields();
+      const currentField = fields.find({ name: this.getOption('field_name') });
+      const defaultHtml = this.getOption('default_html');
+
+      if (!currentField) return { defaultHtml };
+
+      const fieldValue = currentField.get('value');
+      const key = this.getOption('key');
+
+      const displayValue = key ? propertyOf(fieldValue)(key) : fieldValue;
```

Comment:
Oh hmm I just noticed this should be `key.split('.')` and so should phone number.  It should support `some.deeply.nested.key` but without the split it's just a single level down.

I think it's time to DRY this a bit.

I think we should make a standalone function and call it like 
`const value = getFieldValue(this.model.getFields(), this.getOption('field_name'));`

Then we can do with `value` what we need.